rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -2,11 +2,14 @@
 
 from typing import Any, Dict, List, NoReturn, Optional, Text, Tuple, Type
 from tqdm import tqdm
+import os
 
 import rasa.shared.utils.io
-from rasa.nlu.tokenizers.convert_tokenizer import ConveRTTokenizer
+from rasa.core.utils import get_dict_hash
```

Comment:
Please, do not import functions directly but rather the module itself. See https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#70d0247fdd1f48d2aec341fd91fe1dac.